src/universal-app/app/splitter/split-handle.directive.ts

```diff
@@ -0,0 +1,32 @@
+import {Directive, ElementRef, Output} from '@angular/core';
+import {Observable} from 'rxjs/Observable';
+
+import 'rxjs/add/observable/fromEvent';
+import 'rxjs/add/operator/takeUntil';
+import 'rxjs/add/operator/switchMap';
+import 'rxjs/add/operator/map';
+
+@Directive({
+  selector: '[ngxSplitHandle]',
+  host: {
+    class: 'ngx-split-handle',
+    title: 'Drag to resize'
+  }
+})
+export class SplitHandleDirective {
+
+  @Output() drag: Observable<{ x: number, y: number }>;
+
+  constructor(ref: ElementRef) {
+    const getMouseEventPosition = (event: MouseEvent) => ({x: event.movementX, y: event.movementY});
+
+    const mouseup$ = Observable.fromEvent(document, 'mouseup');
```

Comment:
@ardatan - true. I think even better is to use `ref.nativeElement.ownerDocument`. Your thoughts ?

> BTW, even with injection or `ownerDocument` I am still getting `gulp ci:prerender` errors:
```console
11:14:25] 'prerender' errored after 611 ms
[11:14:25] Error: ERROR TypeError: Invalid event target
    at Function.FromEventObservable.setupSubscription (/Users/thomasburleson/Documents/projects/Google/projects/angular-layouts/node_modules/rxjs/observable/FromEventObservable.js:113:19)
    at FromEventObservable._subscribe (/Users/thomasburleson/Documents/projects/Google/projects/angular-layouts/node_modules/rxjs/observable/FromEventObservable.js:135:29)
    at FromEventObservable.Observable._trySubscribe (/Users/thomasburleson/Documents/projects/Google/projects/angular-layouts/node_modules/rxjs/Observable.js:171:25)
    at FromEventObservable.Observable.subscribe (/Users/thomasburleson/Documents/projects/Google/projects/angular-layouts/node_modules/rxjs/Observable.js:159:65)
    at MapOperator.call (/Users/thomasburleson/Documents/projects/Google/projects/angular-layouts/node_modules/rxjs/operator/map.js:54:23)
    at Observable.subscribe (/Users/thomasburleson/Documents/projects/Google/projects/angular-layouts/node_modules/rxjs/Observable.js:156:22)
    at SwitchMapOperator.call (/Users/thomasburleson/Documents/projects/Google/projects/angular-layouts/node_modules/rxjs/operator/switchMap.js:67:23)
    at Observable.subscribe (/Users/thomasburleson/Documents/projects/Google/projects/angular-layouts/node_modules/rxjs/Observable.js:156:22)
    at SplitDirective.ngAfterContentInit (/Users/thomasburleson/Documents/projects/Google/projects/angular-layouts/dist/packages/universal-app/app/splitter/split.directive.js:18:41)
    at callProviderLifecycles (/Users/thomasburleson/Documents/projects/Google/projects/angular-layouts/node_modules/@angular/core/bundles/core.umd.js:11206:18)
```
Any ideas ?